samples/suit/ab/README.rst

```diff
@@ -32,30 +33,279 @@ To perform a SUIT update over UART using the SMP protocol, install SUIT commands
 Overview
 ********
 
-The sample uses one of the following methods to perform a firmware update on the nRF54H20 SoC, featuring Nordic Semiconductor’s implementation of the SUIT procedure:
+The main puprose of this sample is to provide a good starting point for applications that would like to use the "A/B" update method.
+Apart from all configuration parameters, template files and sources, it also introduces a new file structure, that will simplify the development:
+
+* Only a single variant of the application configuration is available.
+* A dedicated file to define the memory map (``sysbuild/nrf54h20dk_nrf54h20_memory_map.dtsi``).
+* A dedicated file to set manifest provisioning configuration (``suit_mpi.conf``).
+* A sample logic for managing manifest variables (``src/device_management.c``).
+* A sample logic for checking the state of manifests (``src/suit_manifest_state.c``).
+* A sample logic for checking the state of in-place updateable components (``src/suit_components_state.c``).
+
+The "A/B" method is based on the assumption that the system is partitioned in such a way that each executable part of the firmware has two memory regions assigned (often referred as ``A`` and ``B`` slots or variants).
+Each of the firmware variants has to be executable from their own memory region, thus every part of the application is going to be (at least) linked twice, once for each slot.
+
+The sample uses several manifest-controlled variables:
```

Comment:
nit: table for manifest variables could improve readability of the section below, something like:
```
| Variable Name      | Values | Description                    |
|-------------------|---------|--------------------------------|
| BOOT_STATUS       | 1     | Indicates boot path ... variant A  |
| BOOT_STATUS       | 2     | Indicates boot path ... variant B |
| BOOT_PREFERENCE   | 1    | Indicates preferred variant ....   |
... and so forth
```